src/widget/wtrackmenu.cpp

```diff
@@ -2049,6 +2049,8 @@ bool WTrackMenu::featureIsEnabled(Feature flag) const {
         return m_pTrackModel->hasCapabilities(TrackModel::Capability::EditMetadata);
     case Feature::SearchRelated:
         return m_pLibrary != nullptr;
+    case Feature::UpdateReplayGain:
+        return m_pTrackModel->hasCapabilities(TrackModel::Capability::EditMetadata);
```

Comment:
This commit has slipped in due to an unwanted rebased or such.
Please rebase your commits to main.

Reply:
it's from #4543 and the only way to build it locally

Reply:
Ok, I have just merged it. So you can rebased this now safely to main.